ui-new/html/minimum-tx.html

```diff
@@ -0,0 +1,11 @@
+<section class="viewport minimum_tx_state cash-in-color" data-tr-section="minimum-tx">
+  <section class="mainv">
+    <section class="header">
+      <img src="images/more-please.svg"/>
+      <h2 class="js-i18n">More, please</h2>
+    </section>
+    <section class="info">
+      <p id="js-i18n-lowest-bill" class="d1">Minimum first bill is</p>
```

Comment:
I don't think this one will work either. js-i18n-lowest-bill will get its contents replaced by `Please insert %s or more.`. Look for lowest-bill on app.js and you will find the piece of code that does that.